react/controls/PlayButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from 'react'
+import { Icon } from 'vtex.store-icons'
```

Comment:
Same thing here, you can import the icons you want to use

Reply:
Same question about the slots @victorhmp and @vitorflg

Reply:
Yep